packages/wallet/src/peer-connect/index.ts

```diff
@@ -0,0 +1,44 @@
+import { WalletAPI } from "../api.js";
+import * as Browser from "../browser/index.js";
+
+type WalletHandler = (walletId: string, wallet: WalletAPI) => void;
+export const mkPeerConnectAdapter = () => {
```

Comment:
this is hard to understand

Reply:
I can add some documentation around it. Have you seen the example to clarify on use?

Reply:
or is there anything in particular that you would need clarification?

Reply:
I would have wrapped this function directly in that file  : 
```
const dAppConnect = new CardanoPeerConnect.DAppPeerConnect({
  dAppInfo: dAppInfo,
  verifyConnection: verifyConnection,
  onApiInject: adapter.adaptApiInject,
  onApiEject: adapter.adaptApiEject,
});
```
I would have kept ` adapter.adaptApiInject` and ` adapter.adaptApiEject` private into this module also

Reply:
Nevertheless, it is good enough like that imo... I would focus on `WalletConnect` and merge this as it is...